Whiskey/Functions/Assert-WhiskeyTaskSkip.ps1

```diff
@@ -0,0 +1,183 @@
+
+function Assert-WhiskeyTaskSkip

```

Comment:
Rename to `Test-WhiskeyTaskSkip`. `Assert` verb means the function throws an exception if it returns `$false`. We just want to return `$false`.